app/views/shared/_navigation_bar.html.erb

```diff
@@ -1,21 +1,39 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-primary justify-content-between" data-turbolinks-animate-persist="true">
   <a class="navbar-brand" href="#"><i class="fas fa-sitemap"></i> Caucus</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggler" aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
   <div class="collapse navbar-collapse" id="navbarToggler" style="justify-content: right;">
-    <% if user_signed_in? && current_user.has_role?(:admin) %>
-    <div class="navbar-nav mr-4">
+    <% if user_signed_in? %>
+
+    <% if current_user.has_role?(:admin) %>
+    <div class="navbar-nav mr-1">
       <%= link_to "Admin", admin_categories_path, class: "nav-link" %>
     </div>
     <% end %>
+
+    <div class="navbar-nav mr-1">
+      <%= link_to("Instructions", instructions_path, class: "nav-link") %>
+    </div>
+
+    <div class="navbar-nav mr-1">
+      <%= image_tag current_user.avatar.variant(resize_to_limit: [25, 25]) if current_user.avatar.attached? %>
+    </div>
```

Comment:
I think users will expect that clicking on the avatar itself will open the dropdown menu; could this be moved inside the dropdown toggle link?